django_x509/base/models.py

```diff
@@ -449,6 +466,13 @@ def get_revoked_certs(self):
                                     validity_start__lte=now,
                                     validity_end__gte=now)
 
+    def renew_ca(self):
+        """
+        Renew the certificate, private key and
+        validity end date of a CA
+        """
+        self._renew()
```

Comment:
I would rename this method to just `renew`, after renewing the CA (calling `super()`), I would loop over the related certificates and call renew on each of them.

Since the second operation could take potentially very long, I would implement it in a separate method, so that in OpenWISP we can then overwrite the method and move it to a celery task if needed.